src/main/java/net/imglib2/algorithm/componenttree/pixellist/PixelListComponentTree.java

```diff
@@ -74,6 +74,7 @@ public final class PixelListComponentTree< T extends Type< T > > implements Comp
 	 * using an {@link ArrayImgFactory} or {@link CellImgFactory} depending on
 	 * input image size.
 	 *
+	 * @implNote op name='create.pixelListComponentTree',type=Function
 	 * @param input
```

Comment:
Like above, should these be ops?

Reply:
Can we use a sub-namespace of `create` here? `create.componentTree.pixelList`, `create.componentTree.mser`, and so on? It would match the package naming. In general, looking at the Java package naming is a good starting point for naming suggestions.

Reply:
Actually... are these really "create" Ops? I think of `create` as merely allocating and returning an object instance, not doing any computation around an image. So this `PixelListComponentTree` shouldn't be a `create` Op at all, since it accepts an existing image as input. `morphology.componentTree.pixelList` maybe? I don't know this space of image algorithms well enough to feel qualified to suggest anything. We could use @tpietzsch's input here, since he wrote these classes. And/or just read the papers about these things.

Reply:
It actually seems to be like the tree best belongs to the `features` namespace, as it is my understanding that component trees themselves have distilled down a 2D function into a tree, where the position is lost. This seems more like feature extraction than morphology.